src/MainWindow.cpp

```diff
@@ -89,61 +89,47 @@ MainWindow::MainWindow(BRect frame) : BWindow(frame, "", B_DOCUMENT_WINDOW, 0)
 	BMenuBar* bar = new BMenuBar("keybar");
 
 	BMenu* menu = new BMenu(B_TRANSLATE("Program"));
```

Comment:
Most programs use either "App" or the app's name "CapitalBe" for this menu. I lately lean toward the latter, but it's your decision.

Reply:
I read some discussion about this, I really like the app icon like Genio uses, but I'm not sure how to implement that at the moment. I opted for "CapitalBe" for now.

Reply:
OK!
From a cursory look at Genio's code, all it needs is to copy its IconMenuItem and then do something similar to https://github.com/Genio-The-Haiku-IDE/Genio/blob/main/src/ui/GenioWindow.cpp#L2871